### DISCUSSION TITLE
Really useful possible additions

### DISCUSSION THREAD

**Discussion by simonmcnair** [moudey/Shell#423] [Ideas]
I currently add reg entries for powershell scripts I have.  It would be useful to be able to add them via this tool.  The main thing it would be able to fix with ease that you cannot do via reg keys is the ability to select more than 20-30-50 items and run an action against them.

currently my powershell scripts are:

'move selected to a timestamped folder' which works badly as each runs sequentially and I don't get a timestamp that works for the 20 selected items without durations inbetween.
'move up' which moves the files a folder upward.
'move to photos' moves the highlighted folders/files to a specific folder
'move to documents' moves the highlighted folders/files to a specific folder
'make txt file' makes a text file with the same basename as the highlighted file.
'move to' moves the selected files to a specified folder

It would be cool to have an 'edit config' option that opens up the config file in notepad too.

I am sure some of these are already possible, but the big win for me is selecting more than 15-20 files and running an action on them as I know regedit can't do this, it has to be via a shell extension.

Might be useful to have a 'run via named ps script' which passes all the selected filenames to a powershell script.

I am new to this, so apologies if it already does the above.
Cheers

**Comment by RubicBG** [comment]
Let's say that everything is achievable, but attention must be paid to one by one, there is no ready-made solution. This is what the Discord channel is for: https://discord.com/channels/1106387012707168318/

**Reply by simonmcnair** [reply]
I think you need to publish an invite to the channel rather than a link, I don't think the link works unless youre' in the channel, which you can only be with an invite ?

**Comment by RubicBG** [comment]
my knowledge of discord is poor, but as far as I know the channel has free access
![image](https://github.com/moudey/Shell/assets/127151533/1b8933c2-a678-413f-978f-3b5d659b9b7a)
https://discord.gg/ys7b7pHV

**Reply by oscarus** [reply]
Invite Invalid 😔
![2024-03-03_14-43-35_chrome](https://github.com/moudey/Shell/assets/10568597/843d3b86-ef74-4176-b2af-3052dbdc8edf)

**Comment by RubicBG** [comment]
https://discord.gg/cH62ySep
![image](https://github.com/moudey/Shell/assets/127151533/f93086c0-ca69-406b-a180-5ad56466d2b4)
only 7 days activ

**Comment by RubicBG** [comment]
https://discord.gg/Mg3tum2sd8
![image](https://github.com/moudey/Shell/assets/127151533/f4fa6b59-2f42-4bbc-b823-accb39c05a59)

**Reply by oscarus** [reply]
I'm in! thank you!
